zone/attack.cpp

```diff
@@ -4302,8 +4302,15 @@ void Mob::CommonDamage(Mob* attacker, int64 &damage, const uint16 spell_id, cons
 				}
 			}
 			else if (skill_used == EQ::skills::SkillKick &&
-				(attacker->GetLevel() > 55 || attacker->IsNPC()) && GetClass() == Class::Warrior) {
-				can_stun = true;
+					attacker->GetClass() == Class::Warrior) {
+				int stun_level;
+				if (attacker->IsClient()) {
+					stun_level = RuleI(Combat, PCKickStunLevel);
+				}
+				else {
+					stun_level = RuleI(Combat, NPCKickStunLevel);
+				}
```

Comment:
Remove the `else` and have this be the default case

```cpp
int stun_level = RuleI(Combat, NPCKickStunLevel);
if (attacker->IsClient()) {
	stun_level = RuleI(Combat, PCKickStunLevel);
}
```

Reply:
Not sure why, as now clients get 2 assignments instead of 1.  But I'm ok with it.  Committing now.